content/swift/concepts/sets/terms/removeAll/removeAll.md

```diff
@@ -0,0 +1,36 @@
+---
+Title: '.removeAll()' # 
+Description: 'Learn how to remove every item from a Set in Swift.' # 
+Subjects: # 
+  - 'iOS'
+  - 'Mobile Development'
+Tags: # Please only use Tags in the tags.md file (https://github.com/Codecademy/docs/blob/main/documentation/tags.md). If that list feels insufficient, feel free to create a new Tag and add it to tags.md in your PR!
+  - 'iOS'
+  - 'SwiftUI'
+  - 'Sets'
+  - 'Xcode'
+CatalogContent: # Please use course/path landing page slugs, rather than linking to individual content items. If listing multiple items, please put the most relevant one first
+  - 'learn-swift'
+  - 'paths/build-ios-apps-with-swiftui'
+---
+
+The **.removeAll()** method is used to remove every value from a [Set](https://www.codecademy.com/resources/docs/swift/sets) in Swift. 
+
+
+## Syntax
+
+```
```

Comment:
This should be a ```pseudo block

Reply:
@SSwiniarski I don't know what "Pseudo blocks" are. I wasn't able to find an answer online either. Could you help me?
I fixed this line as best as I understood.